onnxruntime/contrib_ops/cuda/bert/attention_impl.cu

```diff
@@ -126,30 +212,32 @@ Status QkvToContext(
     if (data.bias == nullptr) {
       // gemm_buffer should be BxSx3xNxH => qkv: 3xBxNxSxH
       ORT_ENFORCE(qk_head_size == v_head_size);
+      ORT_ENFORCE(kv_cache_past_present == 0);
```

Comment:
>ORT_ENFORCE(kv_cache_past_present == 0); [](http://example.com/codeflow?start=6&length=40)

why kv_cache_past_present has to be 0 if bias i nullptr?

Reply:
added logic to support it.